exporter/opensearchexporter/trace_bulk_indexer.go

```diff
@@ -194,6 +194,15 @@ func responseAsError(item opensearchutil.BulkIndexerResponseItem) error {
 	return errors.New(string(errorJSON))
 }
 
+func attributesToMapString(attributes pcommon.Map) map[string]string {
```

Comment:
While this function looks relatively harmless, could we still see a few unit tests added for it?